src/main/resources/payload-template.vm

```diff
@@ -1,12 +1,12 @@
 {
-    "analysisType": {
-        "name": "$name"
-    },
+  "analysisType": {
+    "name": "consensus_sequence"
```

Comment:
for now I'm sticking with this template... but we'll need to make a solid schema for the tsv and finalize this template